multiview/embed/base.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 NeuroData (http://neurodata.io)
```

Comment:
Is this copyright needed here? Rn I don't think it's particularly useful since we already have a license in the repo.

Reply:
I'm going to resolve this since other people are putting this header comment too.